04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,78 @@
+const Product = require("../models/product");
+
+const getAllProductsStatic = async (req, res) => {
+  const products = await Product.find({ price: { $gt: 30 } })
+    .sort("price")
+    .select("name price");
+
+  res.status(200).json({ products, nbHits: products.length });
+};
+
+const getAllProducts = async (req, res) => {
+  const { featured, company, name, sort, fields, numericFilters } = req.query;
+  const queryObject = {};
+
+  if (featured) {
+    queryObject.featured = featured === "true" ? true : false;
```

Comment:
fwiw, `featured === "true"` already resolves to `true` or `false` so the ternary statement isn't necessary. (I know this is how the example shows it...)